src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,94 @@
+package duke.task;
+
+import duke.DukeException;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Event task that inherits from Task
+ */
+public class Event extends Task {
+    protected LocalDate at;
```

Comment:
same issue, naming of variable not easily understandable, would `date` be a better name?